<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>1171</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Connexion">
     <attribute name="title">
      <string>Connexion</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'utilisateur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>190</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot De Passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomU_connexion">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdpU_connexion">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="connexion">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Connexion</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>130</y>
        <width>113</width>
        <height>64</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="sauvegarder_mdp">
         <property name="text">
          <string>Sauvegarder MDP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="afficher_mdp">
         <property name="text">
          <string>Afficher MDP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Création compte</string>
     </attribute>
     <widget class="QLineEdit" name="mdpU_compte">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'utilisateur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomU_compte">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot De Passe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailU_compte">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmer Mot De Passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdpConf_compte">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>260</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="CreerCompte">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Créer compte</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rfid_compte">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>310</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RFID</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>422</x>
        <y>60</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Employe</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher Compte</string>
     </attribute>
     <widget class="QPushButton" name="supprimerU">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>410</y>
        <width>296</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="affichage_nomU">
      <property name="geometry">
       <rect>
        <x>888</x>
        <y>100</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="affichage_mdpU">
      <property name="geometry">
       <rect>
        <x>888</x>
        <y>200</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_users">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>501</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="affichage_emailU">
      <property name="geometry">
       <rect>
        <x>888</x>
        <y>150</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="modifierU">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>360</y>
        <width>296</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_matricule_rec_4">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>150</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_rec_5">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>100</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'utilisateur</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh_users">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom_rec_4">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>200</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="affichage_rfid">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>250</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_nom_rec_5">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>250</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>RFID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_type_rec_6">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="affichage_type">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>50</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Réinitialiser Mode de passe</string>
     </attribute>
     <widget class="QLineEdit" name="mdpU_new1">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nouveau Mot De Passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdpU_new2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmer Nouveau Mot De Passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdpU_old">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ancien Mot De Passe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmer_reset">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomU_reset">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'utilisateur</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
